apps/auth0-rules/package.json

```diff
@@ -20,7 +20,6 @@
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-typescript": "^7.12.7",
     "@babel/runtime-corejs3": "^7.12.5",
-    "@types/asap-hub__auth0-rule": "workspace:*",
```

Comment:
That typings package was very irregular and caused problems with the mapping, I killed it and moved its typings somewhere better